src/backend/routers/para.ts

```diff
@@ -17,8 +17,33 @@ export const para = router({
       return result;
     }),
 
-  getAllParas: authenticatedProcedure.query(async (req) => {
-    const result = await req.ctx.db.selectFrom("user").selectAll().execute();
+  getParaByEmail: authenticatedProcedure
+    .input(z.object({ email: z.string() }))
+    .query(async (req) => {
+      const { email } = req.input;
+
+      const result = await req.ctx.db
+        .selectFrom("user")
+        .where("email", "=", email.toLowerCase())
```

Comment:
🐧 could put this on the zod input schema instead: https://github.com/colinhacks/zod#transform

Reply:
I'm not sure how to do this. Maybe you can show me later, we can do this in another PR, or I can remove this completely since it's not being used anywhere yet.